code/modules/reqs/supplypacks.dm

```diff
@@ -1031,6 +1038,38 @@ MEDICAL
 	contains = list(/obj/item/reagent_containers/glass/bottle/lemoline)
 	cost = 50
 
+/datum/supply_packs/medical/advancedKits
+	name = "Advanced medical packs"
+	notes = "Contains 5 advanced packs of each type and 5 splints."
+	contains = list(
+		/obj/item/stack/medical/advanced/bruise_pack,
+		/obj/item/stack/medical/advanced/bruise_pack,
+		/obj/item/stack/medical/advanced/bruise_pack,
+		/obj/item/stack/medical/advanced/bruise_pack,
+		/obj/item/stack/medical/advanced/bruise_pack,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/advanced/ointment,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		/obj/item/stack/medical/splint,
+		)
```

Comment:
Move the close bracket on 1060 to the end of line 1059 to solve the inconsistent indentation problem that's causing it to fail.

Reply:
OH fuck that was a stupid mistake